crates/client/devnet/src/lib.rs

```diff
@@ -313,7 +316,7 @@ mod tests {
             )
             .unwrap();
 
-        log::debug!("{:?}", backend.get_block_info(&BlockId::Tag(BlockTag::Latest)));
+        tracing::debug!("{:?}", backend.get_block_info(&BlockId::Tag(BlockTag::Latest)));
```

Comment:
I can come up with :
```tracing::debug!("block imported {:?}", backend.get_block_info(&BlockId::Tag(BlockTag::Latest)));```
what do you suggest ?

Reply:
Resolving since this is already committed [here](https://github.com/madara-alliance/madara/pull/348/commits/7da86a8fc2fc83f7da37ba8b78344a66f7863657).